eng/Versions.props

```diff
@@ -9,7 +9,7 @@
     <StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">false</StabilizePackageVersion>
     <DotNetFinalVersionKind Condition="'$(StabilizePackageVersion)' == 'true'">release</DotNetFinalVersionKind>
     <!-- Disable SemVer v2 until after 3.0.0. -->
-    <SemanticVersioningV1 Condition="&#xA;      $(MajorVersion) &lt; 3 or&#xA;      '$(MajorVersion).$(MinorVersion).$(PatchVersion)' == '3.0.0'">true</SemanticVersioningV1>
+    <SemanticVersioningV1 Condition="&#xA;      $(MajorVersion) &lt; 3 or&#xA;      '$(MajorVersion).$(MinorVersion).$(PatchVersion)' == '3.1.0'">true</SemanticVersioningV1>
```

Comment:
Modifying this line breaks the plan, and prevents us from changing to semver v2 until the next preview. #6825

Reply:
Maybe you didn't find https://github.com/dotnet/core-setup/commit/cafcf54a8caccf388715618943cb641fd49f912f#diff-fec826feae04e51c0d94076385408bdc ?